_community_members/tanikp.md

```diff
@@ -0,0 +1,23 @@
+---
+short_name: tanikp
+name: Tanik Pansuriya
+photo: /assets/media/community/members/tanikp.jpg
+title: 'OpenSearch Community Member: Tanik Pansuriya'
+primary_title: Tanik Pansuriya
+breadcrumbs:
+  icon: community
+  items:
+    - title: Community
+      url: /community/index.html
+    - title: Members
+      url: /community/members/index.html
+    - title: 'Tanik Pansuriya&apos;s Profile'
+      url: '/community/members/tanik-pansuriya.html'
+job_title_and_company: 'Software Engineer at AWS'
+personas:
+  - author
+permalink: '/community/members/tanik-pansuriya.html'
+redirect_from: '/authors/tanikp/'
+---
+
+**Tanik** is a Software Development Engineer working at Amazon OpenSearch Service since 2022. At amazon, Tanik primarily focuses on observability of Amazon OpenSearch service.
```

Comment:
```suggestion
**Tanik** is a Software Development Engineer working at Amazon OpenSearch Service since 2022. At Amazon, Tanik primarily focuses on observability of Amazon OpenSearch service.
```